src/Sylius/Bundle/ChannelBundle/Resources/config/app/config.yml

```
@@ -1,6 +1,9 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
+parameters:
+    sylius_channel_env: "%env(default::SYLIUS_CHANNEL)%"
```

This parameter is redundant
```suggestion
    env(SYLIUS_CHANNEL): ~
```